localstack-core/localstack/utils/aws/request_context.py

```diff
@@ -4,13 +4,9 @@
 
 import logging
 import re
-import threading
 from typing import Dict, Optional
 from urllib.parse import urlparse
 
-from flask import request
```

Comment:
🥳 
This is great! Would this allow us now to remove the dependency of flask altogether?
With a quick look I can only find two more usages:
- `aws_responses` where I think that the usage can directly be replaced with `werkzeug`'s `Response` (the superclass of `flask`'s `Response` class)
- Down below in this module in the `patch_moto_request_handling` function, which is maybe not necessary anymore since we don't use the moto server anywhere anymore?

But this should not scope-creep this PR, just an idea for a follow up... 😉